substra/cli/printers.py

```diff
@@ -167,30 +167,35 @@ def _get_list_fields(self):
     def _get_single_fields(self):
         return (self.key_field, ) + self.single_fields
 
-    def print_download_message(self, item):
+    def print_with_profile(self, profile):
+        if profile and profile != 'default':
+            return f'--profile {profile}'
+        return ''
+
+    def print_download_message(self, item, profile=None):
         if self.download_message:
             key_value = self.key_field.get_value(item)
-            print()
-            print(self.download_message)
-            print(f'\tsubstra download {self.asset_name} {key_value}')
+            profile_arg = self.print_with_profile(profile)
+            print('\n' + self.download_message)
```

Comment:
Why the change from `print()` to `print("\n")`? It's almost the same, but now we have sometimes `\n` and sometimes `print()` in the same file.

Reply:
Another solution would be to include the `\n` at the beginning of the download_message string.

Reply:
I think that this is cleaner this way, instead of having an empty print. Adding the `\n` at the beginning of the download_message string means that you have to do that for all of these messages. So for me this is easier (and maybe cleaner) to just put only one `\n` when the message is displayed 😃

Reply:
Sure why not. All my arguments were going to be about taste and not exactly productive. Let's go with it!